CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+### Mapbox Navigation SDK 1.1.1 - October 15, 2020
+
+Visit our [1.0.0 migration guide](https://github.com/mapbox/mapbox-navigation-android/wiki/1.0-Navigation-SDK-Migration-Guide) and the [documentation pages](http://docs.mapbox.com/android/navigation/overview) for more information.
+
+### Changelog
+
+#### Other changes
+* Bump mapbox-common-native version to 7.1.1 [#3651](https://github.com/mapbox/mapbox-navigation-android/pull/3651)
+* Bump mapbox-android-telemetry version to 6.2.1 [#3652](https://github.com/mapbox/mapbox-navigation-android/pull/3652)
+* Bump mapbox-core version to 3.1.0 [#3652](https://github.com/mapbox/mapbox-navigation-android/pull/3652)
```

Comment:
Listing the version bump itself doesn't give end-users information on how it changes the SDK's behavior, we should rather try to focus on the implications of the upgrade. In this case, the user-facing change was:
```
This upgrade resolves an issue with an `OkHttp` dependency mismatch which resulted in narrowing the supported OS versions to API level 21 and higher. Now, the supported range is back to expected API level 19 and higher.
```

Also, it's always better to link to the original PR, rather than the cherry-pick PR.

/cc @mapbox/navigation-android